src/components/Contact.jsx

```diff
@@ -1,21 +1,152 @@
 import React, { Component } from 'react';
-import { Col, Row } from 'react-bootstrap';
-import Header from './Header.jsx';
-import Footer from './Footer.jsx';
+import {
+  Col,
+  Row,
+  FormGroup,
+  FormControl,
+  ControlLabel,
+  Button,
+} from 'react-bootstrap';
 
 class Contact extends Component {
+  // Easy to create an initialState static prop on Contact for resetting the state after the form has been submitted.
+  static initialState = {
+    name: '',
+    message: '',
+    email: '',
+    error: '',
+  };
+
+  state = {
+    ...Contact.initialState,
+    formSubmitted: false,
+  };
+
+  setFormSubmittedState = () => {
+    this.setState({
+      ...Contact.initialState,
+      formSubmitted: true,
+    });
+  }
+
+  handleChange = (event) => {
+    const { name, value } = event.currentTarget;
+
+    this.setState({
+      [name]: value,
+      formSubmitted: false, // Clear out the formSubmitted feedback if the user interacts w/ form.
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+
+    // Run some basic validation.
+    if (!this.state.name) {
+      this.setState({ error: 'Name is required.' });
+
+      return null;
+    }
+
+    // TODO: Add in regex to check for proper email.
+    if (!this.state.email) {
+      this.setState({ error: 'Email is required.' });
+
+      return null;
+    }
+
+    if (!this.state.message) {
+      this.setState({ error: 'Message is required.' });
+
+      return null;
+    }
+
+    // Prepare fetch API options.
+    // API expects application/json
+    const opts = {
+      method: 'POST',
+      body: JSON.stringify(this.state),
+      headers: new Headers({
+        'Content-Type': 'application/json',
+      }),
+    };
+
+    // this url needs to be wherever the server is getting hosted. Obviously localhost:3001 will not work on a production build.
+    const url = process.env.API_URL || 'http://localhost:3001';
```

Comment:
@BlakeGuilloud The domain for the site will be ncreentryhub.org, although we don't have the web server there running ATM. I just set the DNS to point to the AWS instance, but IP is 52.4.230.168. 

I like the idea of putting the domain in the .env.

BTW, I assume this won't hurt, but there will actually be multiple URLs used to access the site. ncreentryhub.org is the main one, but individual counties may access via separate URLs, e.g., we'll eventually point buncombereentryhub.org there and the URL will be used to determine the correct locale.

Does that work?